assets/Scripts/GameManager.cs

```diff
@@ -24,11 +24,14 @@ public class GameManager : MonoBehaviour
         // Use this for initialization
         public void Awake()
         {
-            if (instance == null)
-                instance = this;
-            else if (instance != this)
-                Destroy(gameObject);
-
+            Debug.Log("waking up");
+           // if (instance == null)
+            instance = this;
```

Comment:
THE SUSPICIOUS IF STATEMENT WAS SUSPICIOUS.
but this created a whole new bug......  level does not increment. at all